internal/kafka/internal/api/public/api/openapi.yaml

```diff
@@ -1783,6 +1784,8 @@ components:
           type: boolean
         kafka_storage_size:
           type: string
+        browser_url:
+          type: string
```

Comment:
I am wondering if there is a better way of sending these UI metadata back. 
/cc @miguelsorianod @pb82 this is like the discussion we had in the last backlog refinement about the content schema

Reply:
any follow up on this @machi1990? I wonder if we need to discuss this further before considering merging this PR?

Reply:
There has not been followups on this. 
In my opinion, the answer to the comment I raised above about the other issue maybe applicable here.

Reply:
Resolving this comment in favor of letting the discussion to be progressed on the other issue.